core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1795,18 +1797,17 @@ public String toString() {
   public static final PropertyKey MASTER_EMBEDDED_JOURNAL_CATCHUP_RETRY_WAIT =
       new Builder(Name.MASTER_EMBEDDED_JOURNAL_CATCHUP_RETRY_WAIT)
           .setDefaultValue("1s")
-          .setDescription("Time for embedded journal leader to wait before retrying a catch up.")
+          .setDescription("Time for embedded journal leader to wait before retrying a catch up. "
+              + "This is added to avoid excessive retries when server is not ready.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
   public static final PropertyKey MASTER_EMBEDDED_JOURNAL_ENTRY_SIZE_MAX =
       new Builder(Name.MASTER_EMBEDDED_JOURNAL_ENTRY_SIZE_MAX)
           .setDefaultValue("10MB")
-          .setDescription(String.format(
-              "The maximum single journal entry size allowed to be flushed. "
-              + "This value should be smaller than 30MB. "
-              + "If you update this value, please also update the value of %s ",
-              Name.MASTER_EMBEDDED_JOURNAL_FLUSH_SIZE_MAX))
```

Comment:
Is this comment no longer applicable?

Reply:
I compared with the description found in the docs and picked whichever one had been updated most recently.